CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
+- Added new configuration option `use-pyta-error-messages` to let users choose whether PythonTA should overwrite pylint's error messages.
+
+### Enhancements
```

Comment:
There is a duplicate heading here (possibly the result of a merge conflict). You should combine your new entry with the one below.